packages/core/package.json

```diff
@@ -4218,23 +4218,31 @@
         "clean": "ts-node ../../scripts/clean.ts dist/",
         "copyFiles": "ts-node ./scripts/build/copyFiles.ts",
         "buildScripts": "npm run generateClients && npm run generatePackage && npm run copyFiles",
-        "compile": "npm run clean && npm run buildScripts && tsc -p ./ && webpack --mode development",
-        "compileLite": "tsc -p ./",
-        "watch": "npm run clean && npm run buildScripts && tsc -watch -p ./",
-        "testCompile": "npm run clean && npm run buildScripts && tsc -p ./",
+        "compile": "npm run testCompile && webpack",
+        "compileOnly": "tsc -p ./",
+        "webpackDevelopment": "webpack --mode development",
+        "serveVue": "webpack serve --config-name vue --mode development",
+        "watch": "npm run testCompile -- -- --watch",
+        "testsWebWatch": "npm run clean && npm run buildScripts && npm run webpackDevelopment -- --watch",
+        "testCompile": "npm run clean && npm run buildScripts && npm run compileOnly",
         "test": "npm run testCompile && c8 ts-node ./scripts/test/test.ts",
-        "webCompile": "npm run clean && npm run buildScripts && webpack --config-name web",
-        "webWatch": "npm run webCompile -- --watch",
-        "testWeb": "npm run webCompile && c8 ts-node ./scripts/test/testWeb.ts",
+        "webDevelopmentCompile": "npm run compile -- --mode development",
```

Comment:
Current impl doesn't look related to "web". It's just doing a regular `compile`.

nit: could be named `webDevCompile`, that matches the NPM convention (`devDependencies)`.

Reply:
When I made this I wasn't sure if Web mode would need different configs. But looks like YAGNI. Switching it to `compileDev`